### DISCUSSION TITLE
Episode 214: Advent of Code in BQN (vs Python)

### DISCUSSION THREAD

**Discussion by codereport** [codereport/adsp2#113] [Episode]
Episode Link: https://adspthepodcast.com/2024/12/27/Episode-214.html

Feel free to write comments or ask questions 🙂

**Comment by Abbath** [comment]
https://code.jsoftware.com/wiki/Vocabulary/curlylfco

**Reply by codereport** [reply]
Nice, I always forget that J has the largest set of primitives and many times overlook one.
